modules/settings/assets/js/components/bottom-bar/index.js

```diff
@@ -1,8 +1,28 @@
 import Box from '@elementor/ui/Box';
 import Button from '@elementor/ui/Button';
+import { useSettings, useStorage, useToastNotification } from '@ea11y/hooks';
 import { __ } from '@wordpress/i18n';
 
 export const BottomBar = () => {
+	const { selectedMenu, widgetMenuSettings, hasChanges, setHasChanges } =
+		useSettings();
```

Comment:
```suggestion
	const { 
		selectedMenu,
		widgetMenuSettings,
		hasChanges,
		setHasChanges,
	} = useSettings();
```

Reply:
As per the current ESLint rules and prettier rules, moving these to new lines throws an error. If there was one more item, it would've moved to the new line.

Reply:
offline discussion